### PR TITLE
New DBLD Platforms

### PR DESCRIPTION
**Pull request by therandomstring**
This PR adds three platforms and modifies a fourth. These include:
* **Fedora Rawhide** - for testing purposes, forecasting any eminent breaking changes in Fedora
* **Fedora Latest** - replaces Fedora 40 as an alias
* **AlmaLinux 9** - for building packages for Enterprise Linux 9 compatibles
* **CentOS Stream 9** - for testing purposes, forecasting any eminent breaking changes in EL9

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

File: dbld/build.manifest [hunk 1]
```diff
@@ -43,3 +43,4 @@ ubuntu-mantic		python3
 
 fedora			python3
 almalinux		python3
+centos          python3
```
Comment (kovgeri01) [inline dbld/build.manifest:46]: match the indentation, also it looks like that in this file indentation is done via tabs and not psaces

**Review by kovgeri01 — CHANGES_REQUESTED** [review]

File: dbld/build.manifest (hunk 1 above)
Comment (therandomstring) [reply dbld/build.manifest]: Seems like my IDE had swapped the tabs for spaces... Fixed

**Review by kovgeri01 — APPROVED** [review]
tl;dr: lgtm.

The kira tests are failing because of some new tests which's code is in progress

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build FAILURE
